metadata/metadata.go

```diff
@@ -472,6 +485,10 @@ func (resource *featureVariantResource) ID() ResourceID {
 	}
 }
 
+func (resource *featureVariantResource) Schedule() string {
+	return resource.serialized.UpdateStatus.Schedule.Schedule
```

Comment:
This notation is really strange. Change the second Schedule to Value or something